scripts/gm/init-local.sh

```diff
@@ -77,8 +77,21 @@ gmd gentx $KEY_NAME $STAKING_AMOUNT --chain-id $CHAIN_ID --keyring-backend test
 # collect genesis transactions
 gmd collect-gentxs
 
+# export the Celestia light node's auth token to allow you to submit
+# PayForBlobs to Celestia's data availability network
+# this is for Arabica, if using another network, change the network name
+AUTH_TOKEN=$(celestia light auth write)
```

Comment:
```suggestion
AUTH_TOKEN=$(docker exec $(docker ps -q)  celestia bridge --node.store /home/celestia/bridge/ auth admin)
```

this needs to be for local-celestia-devnet. as that's what init-local is for, this should do it